lib/katello/version.rb

```diff
@@ -1,3 +1,3 @@
 module Katello
-  VERSION = "3.10.0-master".freeze
+  VERSION = "3.11.0-master".freeze
```

Comment:
This needs to just be numbers for `Gem::Dependency` to work I think. For example:

```
irb(main):007:0> Gem::Dependency.new('', ' >= 3.7.0').match?('', '3.10.0')
=> true
irb(main):008:0> Gem::Dependency.new('', ' >= 3.7.0').match?('', '3.10.0.pre.master')
=> false
```